src/Tasks/Microsoft.NET.Build.Tasks.UnitTests/GivenAResolveTargetingPackAssetsTask.cs

```diff
@@ -125,10 +125,9 @@ public void Given_Passing_ResolvedTargetingPacks_A_Different_Language_Parses_Aga
         }
 
         private ResolveTargetingPackAssets InitializeTask(out string mockPackageDirectory,
-            [CallerMemberName] string testName = nameof(GivenAResolvePackageAssetsTask),
-            string identifier = null)
```

Comment:
Why did this get reverted?

Reply:
It's not needed today: https://github.com/dotnet/sdk/pull/19570#discussion_r703831303